src/transforms/metric_to_log.rs

```diff
@@ -408,12 +408,20 @@ mod tests {
         }
     }
 
+    fn event_metadata() -> EventMetadata {
+        let source_type = Arc::new("unit_test_stream".to_owned());
+        let mut event_metadata = EventMetadata::default();
+        event_metadata.set_source_type(source_type);
+        event_metadata
```

Comment:
Small nit - here and elsewhere, you could implement and use `with_source_type` instead.